frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -36,21 +37,13 @@ export default function AppNavbar({
 
           <Navbar.Toggle />
 
-          <>
```

Comment:
why was this removed?

Reply:
i must've done it by accident. everything should be fixed now!